include/boost/geometry/algorithms/detail/buffer/turn_in_piece_visitor.hpp

```diff
@@ -409,7 +409,8 @@ class analyse_turn_wrt_piece
 
         point_type points[4];
 
-        int helper_count = static_cast<int>(piece.robust_ring.size()) - piece.offsetted_count;
+        signed_index_type helper_count = static_cast<signed_index_type>(piece.robust_ring.size())
+                                            - piece.offsetted_count;
```

Comment:
Again, why a value returned by a Range::size() whatever the Range is (probably std::vector) should be assigned to an `int`?